### PR TITLE
set bitsrc as scoped registry with login

### PR DESCRIPTION
**Pull request by amitgilad3**
This change is [Reviewable](https://reviewable.io/reviews/teambit/bit/965)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "preferGlobal": true,
   "scripts": {
     "dev-link": "node ./scripts/establish-dev-link.js $1",
-    "precommit": "lint-staged",
+
```
Comment (GiladShoham) [inline package.json]: revert

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
 
+- when running bit login, also configure bitsrc registry for npm
+
```
Comment (GiladShoham) [inline CHANGELOG.md]: remove one empty line

File: src/cli/commands/public-cmds/login-cmd.js [hunk 3]
```diff
@@ -9,14 +9,16 @@ export default class Login extends Command {
   alias = '';
   opts = [
     ['p', 'port <port>', 'port number to open for localhost server (default 8085)'],
-    ['', 'no-launch-browser', 'do not open a browser for authentication']
+    ['', 'no-launch-browser', 'do not open a browser for authentication'],
+    ['r', 'rc-path <path>', 'path to npmrc file to configure bitsrc registry'],
```
Comment (GiladShoham) [inline src/cli/commands/public-cmds/login-cmd.js]: why rc-path instead of npmrc-path?

File: src/cli/commands/public-cmds/login-cmd.js [hunk 4]
```diff
@@ -9,14 +9,16 @@ export default class Login extends Command {
...
+    ['', 'skip-registry-config [boolean]', 'dont configure bitsrc registry'],
```
Comment (GiladShoham) [inline src/cli/commands/public-cmds/login-cmd.js]: keep in mind that if you put the [boolean] here, i think the user must enter true or false

File: src/driver/driver.js [hunk 5]
```diff
@@ -51,6 +51,13 @@ export default class Driver {
     return driver.getDependencyTree(cwd, consumerPath, fullPaths, bindingPrefix);
   }
 
+
+  // TODO: Improve flow object return type
+  login(token: string, configPath: string, registryUrl: string): Object {
```
Comment (GiladShoham) [inline src/driver/driver.js]: maybe its better to call it npmLogin? or something like this? login is too general

File: src/cli/commands/public-cmds/login-cmd.js (hunk 4 above)
Comment (amitgilad3) [reply src/cli/commands/public-cmds/login-cmd.js]: i check this and the user only needs to write --skip-registry-config and the value is true

**Pull request merged**